front/src/graphql/AuthSettings.graphql

```diff
@@ -0,0 +1,4 @@
+query getAuthSettings {
+  localAuthEnabled
+  ldapAuthEnabled
+}
```

Comment:
This query should probably be moved into [/front/src/graphql/Settings.graphql](https://github.com/TFNS/CTFNote/blob/main/front/src/graphql/Settings.graphql)